src/ResultWriter/AnalysisWriter.cpp

```diff
@@ -200,12 +238,17 @@ void seissol::writer::AnalysisWriter::printAnalysis(double simulationTime) {
         } else {
           MPI_Recv(centerRecv, 3, MPI_DOUBLE,  errLInfRecv[i].rank, i, comm, MPI_STATUS_IGNORE);
         }
-        logInfo(mpi.rank()) << "L1  , var[" << i << "] =\t" << errL1MPI[i];
-        logInfo(mpi.rank()) << "L2  , var[" << i << "] =\t" << std::sqrt(errL2MPI[i]);
-        logInfo(mpi.rank()) << "LInf, var[" << i << "] =\t" << errLInfRecv[i].val
+        const auto errL1 = errL1MPI[i];
+        const auto errL2 = std::sqrt(errL2MPI[i]);
+        const auto errLInf = errLInfRecv[i].val;
+        logInfo(mpi.rank()) << "L1  , var[" << i << "] =\t" << errL1;
+        logInfo(mpi.rank()) << "L2  , var[" << i << "] =\t" << errL2;
+        logInfo(mpi.rank()) << "LInf, var[" << i << "] =\t" << errLInf
             << "at rank " << errLInfRecv[i].rank
             << "\tat [" << centerRecv[0] << ",\t" << centerRecv[1] << ",\t" << centerRecv[2] << "\t]";
-
+        csvWriter.addObservation(std::to_string(i), "L1", errL1);
+        csvWriter.addObservation(std::to_string(i), "L2", errL2);
+        csvWriter.addObservation(std::to_string(i), "LInf", errLInf);
```

Comment:
This is not done for the non mpi part (lines 261-264), but I can not comment on lines not affected by the PR.